dss_client/CMakeLists.txt

```diff
@@ -57,19 +56,22 @@ target_compile_definitions(test_dss PUBLIC "DSS_DEBUG")
 
 target_include_directories(test_dss PUBLIC 
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../nkv-sdk/include>
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
 target_include_directories(${DSS_LIB} PUBLIC 
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-		$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>)
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../nkv-sdk/include>
```

Comment:
Can we do a precheck if nkv-sdk/include is present ? Also, can we pass the nkv-sdk path optionally (see nkv CMake) and default is ../nkv-sdk/include ?

Reply:
The checks are part of scripts/build.sh